### PR TITLE
Add docker as installation option

### PR DESCRIPTION
**Pull request by ilyakaznacheev**
For those who like me don't use Python as a programming language often, and thus don't have a Python environment set up, the container alternative is clean and fast to use.

I've done it for myself, but I'd like to share it with everybody.

*P.S.: thanks for the great tool!*

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM python:3.11-alpine
+
+WORKDIR /extract
+
+COPY . .
+
+RUN pip install -r requirements.txt
+
+RUN pip install qrcode[pil]
```
Comment (scito) [inline Dockerfile:9]: This seems not necessary as it is already in requirements.txt
pil should be the same as Pillow.

File: README.md [hunk 2]
```diff
@@ -118,6 +118,17 @@ Install [devbox](https://github.com/jetpack-io/devbox), which is a wrapper for n
 devbox shell
 ```
 
+### docker
```
Comment (scito) [inline README.md:121]: `### Docker`

**Review by scito — CHANGES_REQUESTED** [review]
Thank you very much for your PR. I'll really appreciate your contribution.
It looks good and useful. I've seen these improvements.
I'll merge it afterwards.

**Comment by ilyakaznacheev** [comment]
Lemme check

File: Dockerfile (hunk 1 above)
Comment (ilyakaznacheev) [reply Dockerfile]: Fixed

File: README.md (hunk 2 above)
Comment (ilyakaznacheev) [reply README.md]: Fixed

**Comment by scito** [comment]
Thank you!

**Pull request merged**